src/app/examples/list/no-shape/list-no-shape-example.component.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../../kirby/scss/utils";
+
+kirby-card {
+  margin: 5px;
```

Comment:
Consider extracting shadow units (offset-x | offset-y | blur-radius | spread-radius) from `src\kirby\scss\base\_functions.scss` into separate scss variables and using them instead.

Reply:
Extracted shadow variables. However calculating the margin based on them is actually not that trivial after all as I see it. Something like:
top = (spread - v_shadow + 0.5*blur)
right = (spread + h_shadow + 0.5*blur)
bottom = (spread + v_shadow + 0.5*blur)
left = (spread - h_shadow + 0.5*blur)

I would prefer it not to be in scope of this PR.